Specs/TypeScript/index.ts

```diff
@@ -121,7 +121,7 @@ property = new ConstantProperty(1);
 property = new TimeIntervalCollectionProperty();
 property = new CompositeProperty();
 property = new SampledProperty(Cartesian3);
-property = new PropertyBag();
+property = {} as PropertyBag;
```

Comment:
Do we want this change? I understand it's hard to prevent this assignment via the type system, but it's not going to work at runtime because expected properties like `propertyNames` will be missing.

Reply:
Oh I see, `PropertyBag` can't be constructed anymore because it's a `type`. Pretty sure that's a problem. It may not be common to construct `PropertyBag`, but it happens. TerriaJS does it.